var/spack/repos/builtin/packages/py-petsc4py/package.py

```diff
@@ -8,7 +8,7 @@
 class PyPetsc4py(PythonPackage):
     """This package provides Python bindings for the PETSc package."""
 
-    homepage = "https://gitlab.com/petsc/petsc4py"
+    homepage = "https://gitlab.com/petsc/petsc"
```

Comment:
@dalcinl  https://pypi.org/project/petsc4py/ uses this url as webpage, so added the same here.

But should we change to:
- https://petsc.org/release/petsc4py/ 

Similarly for slepc4py:
- slepc4py is https://slepc4py.readthedocs.io/en/stable/

Reply:
Yes, I think https://petsc.org/release/petsc4py/ is a better URL for homepage.
The PyPI repository entry gets the URL from petsc4py's `setup.py`, do updating it there should change it the next time we publish a release. 
Same for slepc4py, cc @joseeroman.

Reply:
Thanks! updated both.